rest/handler.go

```diff
@@ -304,6 +304,30 @@ func (h *handler) checkAuth(context *db.DatabaseContext) (err error) {
 
 	}(time.Now())
 
+	// Is there a TLS client cert?
+	var clientCertEmailAddress *string
+	if clientCerts := h.rq.TLS.VerifiedChains; len(clientCerts) > 0 {
```

Comment:
What happens in the case where there is no Client CA set, but a client still sends a cert?

From reading the `crypto/tls` docs, it _looks_ like the initial request handshake will be rejected based on the presence of the default `tls.NoClientCert`, but it's not fully clear if it will be rejected or just ignored?

Reply:
My understanding is that in the TLS handshake the server won't ask the client to send a cert, and a cert sent by the client would cause a fatal error.